executor/executor/heartbeat/__doc__.py

```diff
@@ -0,0 +1,5 @@
+class HeartbeatDoc:
```

Comment:
Do we still use the Heartbeat? I don't think so

Reply:
Yeah, this is something I noticed also, but wasn't so sure. I don't see it being used. Thanks for bringing it up, I'll remove it.